Failing test on GitHub Windows runners: NoClassDefFoundError: org/apache/commons/lang3/ClassUtils

The builds of Dropwizard on GitHub Windows runners have been failing for a while when trying to invoke the Maven archetype as parts of the build.

Examples:

* Java 8/Windows: https://github.com/dropwizard/dropwizard/runs/983747264
* Java 11/Windows: https://github.com/dropwizard/dropwizard/runs/983747297
* Java 14/Windows: https://github.com/dropwizard/dropwizard/runs/983747324

[Complete logs of Java CI #784: `logs_1563.zip`](https://github.com/dropwizard/dropwizard/files/5080963/logs_1563.zip)

```
2020-08-14T07:01:30.8499293Z [INFO] --- maven-invoker-plugin:3.2.1:run (verify) @ java-simple ---
2020-08-14T07:01:30.8720372Z [WARNING] Filtering of parent/child POMs is not supported without cloning the projects
2020-08-14T07:01:30.8840150Z [INFO] Building: ordinary\test-project\pom.xml
[...]
2020-08-14T07:01:50.1878901Z [INFO] [WARNING] Error injecting: org.apache.maven.plugins.javadoc.JavadocJar
2020-08-14T07:01:50.2060587Z [INFO] java.lang.NoClassDefFoundError: org/apache/commons/lang3/ClassUtils
2020-08-14T07:01:50.2061128Z [INFO]     at org.apache.maven.plugins.javadoc.AbstractJavadocMojo.<clinit> (AbstractJavadocMojo.java:205)
2020-08-14T07:01:50.2061685Z [INFO]     at jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance0 (Native Method)
2020-08-14T07:01:50.2062354Z [INFO]     at jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance (NativeConstructorAccessorImpl.java:62)
2020-08-14T07:01:50.2062804Z [INFO]     at jdk.internal.reflect.DelegatingConstructorAccessorImpl.newInstance (DelegatingConstructorAccessorImpl.java:45)
2020-08-14T07:01:50.2063268Z [INFO]     at java.lang.reflect.Constructor.newInstanceWithCaller (Constructor.java:500)
2020-08-14T07:01:50.2063793Z [INFO]     at java.lang.reflect.Constructor.newInstance (Constructor.java:481)
2020-08-14T07:01:50.2064072Z [INFO]     at com.google.inject.internal.DefaultConstructionProxyFactory$ReflectiveProxy.newInstance (DefaultConstructionProxyFactory.java:126)
2020-08-14T07:01:50.2064331Z [INFO]     at com.google.inject.internal.ConstructorInjector.provision (ConstructorInjector.java:114)
2020-08-14T07:01:50.2064801Z [INFO]     at com.google.inject.internal.ConstructorInjector.access$000 (ConstructorInjector.java:32)
2020-08-14T07:01:50.2065042Z [INFO]     at com.google.inject.internal.ConstructorInjector$1.call (ConstructorInjector.java:98)
2020-08-14T07:01:50.2065483Z [INFO]     at com.google.inject.internal.ProvisionListenerStackCallback$Provision.provision (ProvisionListenerStackCallback.java:112)
2020-08-14T07:01:50.2065773Z [INFO]     at com.google.inject.internal.ProvisionListenerStackCallback$Provision.provision (ProvisionListenerStackCallback.java:127)
2020-08-14T07:01:50.2066027Z [INFO]     at com.google.inject.internal.ProvisionListenerStackCallback.provision (ProvisionListenerStackCallback.java:66)
2020-08-14T07:01:50.2066286Z [INFO]     at com.google.inject.internal.ConstructorInjector.construct (ConstructorInjector.java:93)
2020-08-14T07:01:50.2067499Z [INFO]     at com.google.inject.internal.ConstructorBindingImpl$Factory.get (ConstructorBindingImpl.java:306)
2020-08-14T07:01:50.2067850Z [INFO]     at com.google.inject.internal.InjectorImpl$1.get (InjectorImpl.java:1050)
2020-08-14T07:01:50.2068091Z [INFO]     at com.google.inject.internal.InjectorImpl.getInstance (InjectorImpl.java:1086)
2020-08-14T07:01:50.2068331Z [INFO]     at org.eclipse.sisu.space.AbstractDeferredClass.get (AbstractDeferredClass.java:48)
2020-08-14T07:01:50.2068573Z [INFO]     at com.google.inject.internal.ProviderInternalFactory.provision (ProviderInternalFactory.java:85)
2020-08-14T07:01:50.2068824Z [INFO]     at com.google.inject.internal.InternalFactoryToInitializableAdapter.provision (InternalFactoryToInitializableAdapter.java:57)
2020-08-14T07:01:50.2069071Z [INFO]     at com.google.inject.internal.ProviderInternalFactory$1.call (ProviderInternalFactory.java:66)
2020-08-14T07:01:50.2069486Z [INFO]     at com.google.inject.internal.ProvisionListenerStackCallback$Provision.provision (ProvisionListenerStackCallback.java:112)
2020-08-14T07:01:50.2069728Z [INFO]     at com.google.inject.internal.ProvisionListenerStackCallback$Provision.provision (ProvisionListenerStackCallback.java:127)
2020-08-14T07:01:50.2069972Z [INFO]     at com.google.inject.internal.ProvisionListenerStackCallback.provision (ProvisionListenerStackCallback.java:66)
2020-08-14T07:01:50.2070204Z [INFO]     at com.google.inject.internal.ProviderInternalFactory.circularGet (ProviderInternalFactory.java:61)
2020-08-14T07:01:50.2070455Z [INFO]     at com.google.inject.internal.InternalFactoryToInitializableAdapter.get (InternalFactoryToInitializableAdapter.java:47)
2020-08-14T07:01:50.2070687Z [INFO]     at com.google.inject.internal.InjectorImpl$1.get (InjectorImpl.java:1050)
2020-08-14T07:01:50.2070909Z [INFO]     at org.eclipse.sisu.inject.Guice4$1.get (Guice4.java:162)
2020-08-14T07:01:50.2071359Z [INFO]     at org.eclipse.sisu.inject.LazyBeanEntry.getValue (LazyBeanEntry.java:81)
2020-08-14T07:01:50.2071675Z [INFO]     at org.eclipse.sisu.plexus.LazyPlexusBean.getValue (LazyPlexusBean.java:51)
2020-08-14T07:01:50.2071925Z [INFO]     at org.codehaus.plexus.DefaultPlexusContainer.lookup (DefaultPlexusContainer.java:263)
2020-08-14T07:01:50.2072159Z [INFO]     at org.codehaus.plexus.DefaultPlexusContainer.lookup (DefaultPlexusContainer.java:255)
2020-08-14T07:01:50.2072394Z [INFO]     at org.apache.maven.plugin.internal.DefaultMavenPluginManager.getConfiguredMojo (DefaultMavenPluginManager.java:520)
2020-08-14T07:01:50.2072636Z [INFO]     at org.apache.maven.plugin.DefaultBuildPluginManager.executeMojo (DefaultBuildPluginManager.java:124)
2020-08-14T07:01:50.2072866Z [INFO]     at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:210)
2020-08-14T07:01:50.2073093Z [INFO]     at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:156)
2020-08-14T07:01:50.2073341Z [INFO]     at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:148)
2020-08-14T07:01:50.2204094Z [INFO]     at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject (LifecycleModuleBuilder.java:117)
2020-08-14T07:01:50.2204458Z [INFO]     at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject (LifecycleModuleBuilder.java:81)
2020-08-14T07:01:50.2204757Z [INFO]     at org.apache.maven.lifecycle.internal.builder.singlethreaded.SingleThreadedBuilder.build (SingleThreadedBuilder.java:56)
2020-08-14T07:01:50.2205004Z [INFO]     at org.apache.maven.lifecycle.internal.LifecycleStarter.execute (LifecycleStarter.java:128)
2020-08-14T07:01:50.2205239Z [INFO]     at org.apache.maven.DefaultMaven.doExecute (DefaultMaven.java:305)
2020-08-14T07:01:50.2205469Z [INFO]     at org.apache.maven.DefaultMaven.doExecute (DefaultMaven.java:192)
2020-08-14T07:01:50.2205690Z [INFO]     at org.apache.maven.DefaultMaven.execute (DefaultMaven.java:105)
2020-08-14T07:01:50.2206387Z [INFO]     at org.apache.maven.cli.MavenCli.execute (MavenCli.java:957)
2020-08-14T07:01:50.2206707Z [INFO]     at org.apache.maven.cli.MavenCli.doMain (MavenCli.java:289)
2020-08-14T07:01:50.2206930Z [INFO]     at org.apache.maven.cli.MavenCli.main (MavenCli.java:193)
2020-08-14T07:01:50.2207162Z [INFO]     at jdk.internal.reflect.NativeMethodAccessorImpl.invoke0 (Native Method)
2020-08-14T07:01:50.2207749Z [INFO]     at jdk.internal.reflect.NativeMethodAccessorImpl.invoke (NativeMethodAccessorImpl.java:62)
2020-08-14T07:01:50.2208014Z [INFO]     at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke (DelegatingMethodAccessorImpl.java:43)
2020-08-14T07:01:50.2208462Z [INFO]     at java.lang.reflect.Method.invoke (Method.java:564)
2020-08-14T07:01:50.2208716Z [INFO]     at org.codehaus.plexus.classworlds.launcher.Launcher.launchEnhanced (Launcher.java:282)
2020-08-14T07:01:50.2208971Z [INFO]     at org.codehaus.plexus.classworlds.launcher.Launcher.launch (Launcher.java:225)
2020-08-14T07:01:50.2209258Z [INFO]     at org.codehaus.plexus.classworlds.launcher.Launcher.mainWithExitCode (Launcher.java:406)
2020-08-14T07:01:50.2209519Z [INFO]     at org.codehaus.plexus.classworlds.launcher.Launcher.main (Launcher.java:347)
2020-08-14T07:01:50.2209773Z [INFO] Caused by: java.lang.ClassNotFoundException: org.apache.commons.lang3.ClassUtils
2020-08-14T07:01:50.2210061Z [INFO]     at org.codehaus.plexus.classworlds.strategy.SelfFirstStrategy.loadClass (SelfFirstStrategy.java:50)
2020-08-14T07:01:50.2210330Z [INFO]     at org.codehaus.plexus.classworlds.realm.ClassRealm.unsynchronizedLoadClass (ClassRealm.java:271)
2020-08-14T07:01:50.2210591Z [INFO]     at org.codehaus.plexus.classworlds.realm.ClassRealm.loadClass (ClassRealm.java:247)
2020-08-14T07:01:50.2210848Z [INFO]     at org.codehaus.plexus.classworlds.realm.ClassRealm.loadClass (ClassRealm.java:239)
2020-08-14T07:01:50.2211158Z [INFO]     at org.apache.maven.plugins.javadoc.AbstractJavadocMojo.<clinit> (AbstractJavadocMojo.java:205)
2020-08-14T07:01:50.2212012Z [INFO]     at jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance0 (Native Method)
2020-08-14T07:01:50.2212314Z [INFO]     at jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance (NativeConstructorAccessorImpl.java:62)
2020-08-14T07:01:50.2212594Z [INFO]     at jdk.internal.reflect.DelegatingConstructorAccessorImpl.newInstance (DelegatingConstructorAccessorImpl.java:45)
2020-08-14T07:01:50.2212877Z [INFO]     at java.lang.reflect.Constructor.newInstanceWithCaller (Constructor.java:500)
2020-08-14T07:01:50.2214146Z [INFO]     at java.lang.reflect.Constructor.newInstance (Constructor.java:481)
2020-08-14T07:01:50.2214440Z [INFO]     at com.google.inject.internal.DefaultConstructionProxyFactory$ReflectiveProxy.newInstance (DefaultConstructionProxyFactory.java:126)
2020-08-14T07:01:50.2214707Z [INFO]     at com.google.inject.internal.ConstructorInjector.provision (ConstructorInjector.java:114)
2020-08-14T07:01:50.2214988Z [INFO]     at com.google.inject.internal.ConstructorInjector.access$000 (ConstructorInjector.java:32)
2020-08-14T07:01:50.2215252Z [INFO]     at com.google.inject.internal.ConstructorInjector$1.call (ConstructorInjector.java:98)
2020-08-14T07:01:50.2215523Z [INFO]     at com.google.inject.internal.ProvisionListenerStackCallback$Provision.provision (ProvisionListenerStackCallback.java:112)
2020-08-14T07:01:50.2215799Z [INFO]     at com.google.inject.internal.ProvisionListenerStackCallback$Provision.provision (ProvisionListenerStackCallback.java:127)
2020-08-14T07:01:50.2216077Z [INFO]     at com.google.inject.internal.ProvisionListenerStackCallback.provision (ProvisionListenerStackCallback.java:66)
2020-08-14T07:01:50.2216358Z [INFO]     at com.google.inject.internal.ConstructorInjector.construct (ConstructorInjector.java:93)
2020-08-14T07:01:50.2216625Z [INFO]     at com.google.inject.internal.ConstructorBindingImpl$Factory.get (ConstructorBindingImpl.java:306)
2020-08-14T07:01:50.2216894Z [INFO]     at com.google.inject.internal.InjectorImpl$1.get (InjectorImpl.java:1050)
2020-08-14T07:01:50.2217605Z [INFO]     at com.google.inject.internal.InjectorImpl.getInstance (InjectorImpl.java:1086)
2020-08-14T07:01:50.2354597Z [INFO]     at org.eclipse.sisu.space.AbstractDeferredClass.get (AbstractDeferredClass.java:48)
2020-08-14T07:01:50.2360732Z [INFO]     at com.google.inject.internal.ProviderInternalFactory.provision (ProviderInternalFactory.java:85)
2020-08-14T07:01:50.2361227Z [INFO]     at com.google.inject.internal.InternalFactoryToInitializableAdapter.provision (InternalFactoryToInitializableAdapter.java:57)
2020-08-14T07:01:50.2361677Z [INFO]     at com.google.inject.internal.ProviderInternalFactory$1.call (ProviderInternalFactory.java:66)
2020-08-14T07:01:50.2361952Z [INFO]     at com.google.inject.internal.ProvisionListenerStackCallback$Provision.provision (ProvisionListenerStackCallback.java:112)
2020-08-14T07:01:50.2362460Z [INFO]     at com.google.inject.internal.ProvisionListenerStackCallback$Provision.provision (ProvisionListenerStackCallback.java:127)
2020-08-14T07:01:50.2362787Z [INFO]     at com.google.inject.internal.ProvisionListenerStackCallback.provision (ProvisionListenerStackCallback.java:66)
2020-08-14T07:01:50.2363054Z [INFO]     at com.google.inject.internal.ProviderInternalFactory.circularGet (ProviderInternalFactory.java:61)
2020-08-14T07:01:50.2363336Z [INFO]     at com.google.inject.internal.InternalFactoryToInitializableAdapter.get (InternalFactoryToInitializableAdapter.java:47)
2020-08-14T07:01:50.2363589Z [INFO]     at com.google.inject.internal.InjectorImpl$1.get (InjectorImpl.java:1050)
2020-08-14T07:01:50.2363833Z [INFO]     at org.eclipse.sisu.inject.Guice4$1.get (Guice4.java:162)
2020-08-14T07:01:50.2364101Z [INFO]     at org.eclipse.sisu.inject.LazyBeanEntry.getValue (LazyBeanEntry.java:81)
2020-08-14T07:01:50.2364349Z [INFO]     at org.eclipse.sisu.plexus.LazyPlexusBean.getValue (LazyPlexusBean.java:51)
2020-08-14T07:01:50.2364801Z [INFO]     at org.codehaus.plexus.DefaultPlexusContainer.lookup (DefaultPlexusContainer.java:263)
2020-08-14T07:01:50.2365058Z [INFO]     at org.codehaus.plexus.DefaultPlexusContainer.lookup (DefaultPlexusContainer.java:255)
2020-08-14T07:01:50.2365474Z [INFO]     at org.apache.maven.plugin.internal.DefaultMavenPluginManager.getConfiguredMojo (DefaultMavenPluginManager.java:520)
2020-08-14T07:01:50.2365880Z [INFO]     at org.apache.maven.plugin.DefaultBuildPluginManager.executeMojo (DefaultBuildPluginManager.java:124)
2020-08-14T07:01:50.2366115Z [INFO]     at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:210)
2020-08-14T07:01:50.2366345Z [INFO]     at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:156)
2020-08-14T07:01:50.2366571Z [INFO]     at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:148)
2020-08-14T07:01:50.2366809Z [INFO]     at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject (LifecycleModuleBuilder.java:117)
2020-08-14T07:01:50.2367071Z [INFO]     at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject (LifecycleModuleBuilder.java:81)
2020-08-14T07:01:50.2367326Z [INFO]     at org.apache.maven.lifecycle.internal.builder.singlethreaded.SingleThreadedBuilder.build (SingleThreadedBuilder.java:56)
2020-08-14T07:01:50.2367758Z [INFO]     at org.apache.maven.lifecycle.internal.LifecycleStarter.execute (LifecycleStarter.java:128)
2020-08-14T07:01:50.2368006Z [INFO]     at org.apache.maven.DefaultMaven.doExecute (DefaultMaven.java:305)
2020-08-14T07:01:50.2368246Z [INFO]     at org.apache.maven.DefaultMaven.doExecute (DefaultMaven.java:192)
2020-08-14T07:01:50.2368492Z [INFO]     at org.apache.maven.DefaultMaven.execute (DefaultMaven.java:105)
2020-08-14T07:01:50.2368723Z [INFO]     at org.apache.maven.cli.MavenCli.execute (MavenCli.java:957)
2020-08-14T07:01:50.2368954Z [INFO]     at org.apache.maven.cli.MavenCli.doMain (MavenCli.java:289)
2020-08-14T07:01:50.2369359Z [INFO]     at org.apache.maven.cli.MavenCli.main (MavenCli.java:193)
2020-08-14T07:01:50.2370927Z [INFO]     at jdk.internal.reflect.NativeMethodAccessorImpl.invoke0 (Native Method)
2020-08-14T07:01:50.2371405Z [INFO]     at jdk.internal.reflect.NativeMethodAccessorImpl.invoke (NativeMethodAccessorImpl.java:62)
2020-08-14T07:01:50.2371700Z [INFO]     at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke (DelegatingMethodAccessorImpl.java:43)
2020-08-14T07:01:50.2371952Z [INFO]     at java.lang.reflect.Method.invoke (Method.java:564)
2020-08-14T07:01:50.2372199Z [INFO]     at org.codehaus.plexus.classworlds.launcher.Launcher.launchEnhanced (Launcher.java:282)
2020-08-14T07:01:50.2372447Z [INFO]     at org.codehaus.plexus.classworlds.launcher.Launcher.launch (Launcher.java:225)
2020-08-14T07:01:50.2372701Z [INFO]     at org.codehaus.plexus.classworlds.launcher.Launcher.mainWithExitCode (Launcher.java:406)
2020-08-14T07:01:50.2372953Z [INFO]     at org.codehaus.plexus.classworlds.launcher.Launcher.main (Launcher.java:347)
2020-08-14T07:01:50.2373217Z [INFO] [INFO] ------------------------------------------------------------------------
2020-08-14T07:01:50.2373448Z [INFO] [INFO] BUILD FAILURE
2020-08-14T07:01:50.2373917Z [INFO] [INFO] ------------------------------------------------------------------------
2020-08-14T07:01:50.2374141Z [INFO] [INFO] Total time:  17.278 s
2020-08-14T07:01:50.2374547Z [INFO] [INFO] Finished at: 2020-08-14T07:01:50Z
2020-08-14T07:01:50.2509173Z [INFO] [INFO] ------------------------------------------------------------------------
2020-08-14T07:01:50.2561788Z [INFO] [ERROR] Failed to execute goal org.apache.maven.plugins:maven-javadoc-plugin:3.2.0:jar (attach-javadocs) on project test-project: Execution attach-javadocs of goal org.apache.maven.plugins:maven-javadoc-plugin:3.2.0:jar failed: A required class was missing while executing org.apache.maven.plugins:maven-javadoc-plugin:3.2.0:jar: org/apache/commons/lang3/ClassUtils
2020-08-14T07:01:50.2563438Z [INFO] [ERROR] -----------------------------------------------------
2020-08-14T07:01:50.2575567Z [INFO] [ERROR] realm =    plugin>org.apache.maven.plugins:maven-javadoc-plugin:3.2.0
2020-08-14T07:01:50.3821387Z [INFO] [ERROR] strategy = org.codehaus.plexus.classworlds.strategy.SelfFirstStrategy
2020-08-14T07:01:50.3822064Z [INFO] [ERROR] urls[0] = file:/D:/a/dropwizard/dropwizard/dropwizard-archetypes/java-simple/target/it/repo/org/apache/maven/plugins/maven-javadoc-plugin/3.2.0/maven-javadoc-plugin-3.2.0.jar
2020-08-14T07:01:50.3822394Z [INFO] [ERROR] urls[1] = file:/D:/a/dropwizard/dropwizard/dropwizard-archetypes/java-simple/target/it/repo/org/sonatype/aether/aether-util/1.13.1/aether-util-1.13.1.jar
2020-08-14T07:01:50.3822676Z [INFO] [ERROR] urls[2] = file:/D:/a/dropwizard/dropwizard/dropwizard-archetypes/java-simple/target/it/repo/org/sonatype/sisu/sisu-inject-bean/1.4.2/sisu-inject-bean-1.4.2.jar
2020-08-14T07:01:50.3822944Z [INFO] [ERROR] urls[3] = file:/D:/a/dropwizard/dropwizard/dropwizard-archetypes/java-simple/target/it/repo/org/sonatype/sisu/sisu-guice/2.1.7/sisu-guice-2.1.7-noaop.jar
2020-08-14T07:01:50.3823243Z [INFO] [ERROR] urls[4] = file:/D:/a/dropwizard/dropwizard/dropwizard-archetypes/java-simple/target/it/repo/org/codehaus/plexus/plexus-interpolation/1.14/plexus-interpolation-1.14.jar
2020-08-14T07:01:50.3823540Z [INFO] [ERROR] urls[5] = file:/D:/a/dropwizard/dropwizard/dropwizard-archetypes/java-simple/target/it/repo/org/codehaus/plexus/plexus-component-annotations/2.0.0/plexus-component-annotations-2.0.0.jar
2020-08-14T07:01:50.3823968Z [INFO] [ERROR] urls[6] = file:/D:/a/dropwizard/dropwizard/dropwizard-archetypes/java-simple/target/it/repo/org/sonatype/plexus/plexus-sec-dispatcher/1.3/plexus-sec-dispatcher-1.3.jar
2020-08-14T07:01:50.3824217Z [INFO] [ERROR] urls[7] = file:/D:/a/dropwizard/dropwizard/dropwizard-archetypes/java-simple/target/it/repo/org/sonatype/plexus/plexus-cipher/1.4/plexus-cipher-1.4.jar
2020-08-14T07:01:50.3825352Z [INFO] [ERROR] urls[8] = file:/D:/a/dropwizard/dropwizard/dropwizard-archetypes/java-simple/target/it/repo/org/apache/maven/reporting/maven-reporting-api/3.0/maven-reporting-api-3.0.jar
2020-08-14T07:01:50.3825727Z [INFO] [ERROR] urls[9] = file:/D:/a/dropwizard/dropwizard/dropwizard-archetypes/java-simple/target/it/repo/org/apache/maven/maven-archiver/3.5.0/maven-archiver-3.5.0.jar
2020-08-14T07:01:50.3825988Z [INFO] [ERROR] urls[10] = file:/D:/a/dropwizard/dropwizard/dropwizard-archetypes/java-simple/target/it/repo/org/apache/maven/shared/maven-shared-utils/3.2.1/maven-shared-utils-3.2.1.jar
2020-08-14T07:01:50.3826249Z [INFO] [ERROR] urls[11] = file:/D:/a/dropwizard/dropwizard/dropwizard-archetypes/java-simple/target/it/repo/org/apache/maven/shared/maven-invoker/3.0.0/maven-invoker-3.0.0.jar
2020-08-14T07:01:50.3826537Z [INFO] [ERROR] urls[12] = file:/D:/a/dropwizard/dropwizard/dropwizard-archetypes/java-simple/target/it/repo/org/apache/maven/shared/maven-common-artifact-filters/3.0.0/maven-common-artifact-filters-3.0.0.jar
2020-08-14T07:01:50.3826819Z [INFO] [ERROR] urls[13] = file:/D:/a/dropwizard/dropwizard/dropwizard-archetypes/java-simple/target/it/repo/org/apache/maven/shared/maven-artifact-transfer/0.10.1/maven-artifact-transfer-0.10.1.jar
2020-08-14T07:01:50.3827243Z [INFO] [ERROR] urls[14] = file:/D:/a/dropwizard/dropwizard/dropwizard-archetypes/java-simple/target/it/repo/commons-codec/commons-codec/1.11/commons-codec-1.11.jar
2020-08-14T07:01:50.3827493Z [INFO] [ERROR] urls[15] = file:/D:/a/dropwizard/dropwizard/dropwizard-archetypes/java-simple/target/it/repo/org/apache/maven/doxia/doxia-sink-api/1.7/doxia-sink-api-1.7.jar
2020-08-14T07:01:50.3827742Z [INFO] [ERROR] urls[16] = file:/D:/a/dropwizard/dropwizard/dropwizard-archetypes/java-simple/target/it/repo/org/apache/maven/doxia/doxia-logging-api/1.7/doxia-logging-api-1.7.jar
2020-08-14T07:01:50.3827996Z [INFO] [ERROR] urls[17] = file:/D:/a/dropwizard/dropwizard/dropwizard-archetypes/java-simple/target/it/repo/org/apache/maven/doxia/doxia-site-renderer/1.9.2/doxia-site-renderer-1.9.2.jar
2020-08-14T07:01:50.3828732Z [INFO] [ERROR] urls[18] = file:/D:/a/dropwizard/dropwizard/dropwizard-archetypes/java-simple/target/it/repo/org/apache/maven/doxia/doxia-core/1.9.1/doxia-core-1.9.1.jar
2020-08-14T07:01:50.3872989Z [INFO] [ERROR] urls[19] = file:/D:/a/dropwizard/dropwizard/dropwizard-archetypes/java-simple/target/it/repo/org/apache/maven/doxia/doxia-decoration-model/1.9.2/doxia-decoration-model-1.9.2.jar
2020-08-14T07:01:50.3873353Z [INFO] [ERROR] urls[20] = file:/D:/a/dropwizard/dropwizard/dropwizard-archetypes/java-simple/target/it/repo/org/apache/maven/doxia/doxia-skin-model/1.9.2/doxia-skin-model-1.9.2.jar
2020-08-14T07:01:50.3873654Z [INFO] [ERROR] urls[21] = file:/D:/a/dropwizard/dropwizard/dropwizard-archetypes/java-simple/target/it/repo/org/apache/maven/doxia/doxia-module-xhtml/1.9.1/doxia-module-xhtml-1.9.1.jar
2020-08-14T07:01:50.3873965Z [INFO] [ERROR] urls[22] = file:/D:/a/dropwizard/dropwizard/dropwizard-archetypes/java-simple/target/it/repo/org/apache/maven/doxia/doxia-module-xhtml5/1.9.1/doxia-module-xhtml5-1.9.1.jar
2020-08-14T07:01:50.3874271Z [INFO] [ERROR] urls[23] = file:/D:/a/dropwizard/dropwizard/dropwizard-archetypes/java-simple/target/it/repo/org/codehaus/plexus/plexus-i18n/1.0-beta-10/plexus-i18n-1.0-beta-10.jar
2020-08-14T07:01:50.3874551Z [INFO] [ERROR] urls[24] = file:/D:/a/dropwizard/dropwizard/dropwizard-archetypes/java-simple/target/it/repo/junit/junit/3.8.1/junit-3.8.1.jar
2020-08-14T07:01:50.3874823Z [INFO] [ERROR] urls[25] = file:/D:/a/dropwizard/dropwizard/dropwizard-archetypes/java-simple/target/it/repo/org/codehaus/plexus/plexus-velocity/1.2/plexus-velocity-1.2.jar
2020-08-14T07:01:50.3875100Z [INFO] [ERROR] urls[26] = file:/D:/a/dropwizard/dropwizard/dropwizard-archetypes/java-simple/target/it/repo/org/apache/velocity/velocity/1.7/velocity-1.7.jar
2020-08-14T07:01:50.3875537Z [INFO] [ERROR] urls[27] = file:/D:/a/dropwizard/dropwizard/dropwizard-archetypes/java-simple/target/it/repo/commons-lang/commons-lang/2.4/commons-lang-2.4.jar
2020-08-14T07:01:50.3876240Z [INFO] [ERROR] urls[28] = file:/D:/a/dropwizard/dropwizard/dropwizard-archetypes/java-simple/target/it/repo/org/apache/velocity/velocity-tools/2.0/velocity-tools-2.0.jar
2020-08-14T07:01:50.3877024Z [INFO] [ERROR] urls[29] = file:/D:/a/dropwizard/dropwizard/dropwizard-archetypes/java-simple/target/it/repo/commons-beanutils/commons-beanutils/1.7.0/commons-beanutils-1.7.0.jar
2020-08-14T07:01:50.3877313Z [INFO] [ERROR] urls[30] = file:/D:/a/dropwizard/dropwizard/dropwizard-archetypes/java-simple/target/it/repo/commons-digester/commons-digester/1.8/commons-digester-1.8.jar
2020-08-14T07:01:50.3877572Z [INFO] [ERROR] urls[31] = file:/D:/a/dropwizard/dropwizard/dropwizard-archetypes/java-simple/target/it/repo/commons-chain/commons-chain/1.1/commons-chain-1.1.jar
2020-08-14T07:01:50.3877833Z [INFO] [ERROR] urls[32] = file:/D:/a/dropwizard/dropwizard/dropwizard-archetypes/java-simple/target/it/repo/dom4j/dom4j/1.1/dom4j-1.1.jar
2020-08-14T07:01:50.3878078Z [INFO] [ERROR] urls[33] = file:/D:/a/dropwizard/dropwizard/dropwizard-archetypes/java-simple/target/it/repo/oro/oro/2.0.8/oro-2.0.8.jar
2020-08-14T07:01:50.3878347Z [INFO] [ERROR] urls[34] = file:/D:/a/dropwizard/dropwizard/dropwizard-archetypes/java-simple/target/it/repo/commons-collections/commons-collections/3.2.2/commons-collections-3.2.2.jar
2020-08-14T07:01:50.3878606Z [INFO] [ERROR] urls[35] = file:/D:/a/dropwizard/dropwizard/dropwizard-archetypes/java-simple/target/it/repo/org/apache/commons/commons-lang3/3.5/commons-lang3-3.5.jar
2020-08-14T07:01:50.3878859Z [INFO] [ERROR] urls[36] = file:/D:/a/dropwizard/dropwizard/dropwizard-archetypes/java-simple/target/it/repo/commons-io/commons-io/2.5/commons-io-2.5.jar
2020-08-14T07:01:50.3879129Z [INFO] [ERROR] urls[37] = file:/D:/a/dropwizard/dropwizard/dropwizard-archetypes/java-simple/target/it/repo/org/apache/httpcomponents/httpclient/4.5.8/httpclient-4.5.8.jar
2020-08-14T07:01:50.3879394Z [INFO] [ERROR] urls[38] = file:/D:/a/dropwizard/dropwizard/dropwizard-archetypes/java-simple/target/it/repo/org/apache/httpcomponents/httpcore/4.4.11/httpcore-4.4.11.jar
2020-08-14T07:01:50.3880586Z [INFO] [ERROR] urls[39] = file:/D:/a/dropwizard/dropwizard/dropwizard-archetypes/java-simple/target/it/repo/commons-logging/commons-logging/1.2/commons-logging-1.2.jar
2020-08-14T07:01:50.3880874Z [INFO] [ERROR] urls[40] = file:/D:/a/dropwizard/dropwizard/dropwizard-archetypes/java-simple/target/it/repo/com/thoughtworks/qdox/qdox/2.0-M10/qdox-2.0-M10.jar
2020-08-14T07:01:50.3881153Z [INFO] [ERROR] urls[41] = file:/D:/a/dropwizard/dropwizard/dropwizard-archetypes/java-simple/target/it/repo/org/codehaus/plexus/plexus-java/1.0.5/plexus-java-1.0.5.jar
2020-08-14T07:01:50.3881444Z [INFO] [ERROR] urls[42] = file:/D:/a/dropwizard/dropwizard/dropwizard-archetypes/java-simple/target/it/repo/org/ow2/asm/asm/7.2/asm-7.2.jar
2020-08-14T07:01:50.3887873Z [INFO] [ERROR] urls[43] = file:/D:/a/dropwizard/dropwizard/dropwizard-archetypes/java-simple/target/it/repo/org/codehaus/plexus/plexus-archiver/4.2.1/plexus-archiver-4.2.1.jar
2020-08-14T07:01:50.3888181Z [INFO] [ERROR] urls[44] = file:/D:/a/dropwizard/dropwizard/dropwizard-archetypes/java-simple/target/it/repo/org/apache/commons/commons-compress/1.19/commons-compress-1.19.jar
2020-08-14T07:01:50.3888447Z [INFO] [ERROR] urls[45] = file:/D:/a/dropwizard/dropwizard/dropwizard-archetypes/java-simple/target/it/repo/org/iq80/snappy/snappy/0.4/snappy-0.4.jar
2020-08-14T07:01:50.3888712Z [INFO] [ERROR] urls[46] = file:/D:/a/dropwizard/dropwizard/dropwizard-archetypes/java-simple/target/it/repo/org/tukaani/xz/1.8/xz-1.8.jar
2020-08-14T07:01:50.3888966Z [INFO] [ERROR] urls[47] = file:/D:/a/dropwizard/dropwizard/dropwizard-archetypes/java-simple/target/it/repo/org/codehaus/plexus/plexus-io/3.2.0/plexus-io-3.2.0.jar
2020-08-14T07:01:50.3889223Z [INFO] [ERROR] urls[48] = file:/D:/a/dropwizard/dropwizard/dropwizard-archetypes/java-simple/target/it/repo/org/codehaus/plexus/plexus-utils/3.3.0/plexus-utils-3.3.0.jar
2020-08-14T07:01:50.3889597Z [INFO] [ERROR] urls[49] = file:/D:/a/dropwizard/dropwizard/dropwizard-archetypes/java-simple/target/it/repo/org/codehaus/plexus/plexus-interactivity-api/1.0-alpha-6/plexus-interactivity-api-1.0-alpha-6.jar
2020-08-14T07:01:50.3889871Z [INFO] [ERROR] Number of foreign imports: 1
2020-08-14T07:01:50.3890089Z [INFO] [ERROR] import: Entry[import  from realm ClassRealm[maven.api, parent: null]]
2020-08-14T07:01:50.3890292Z [INFO] [ERROR] 
2020-08-14T07:01:50.3890509Z [INFO] [ERROR] -----------------------------------------------------: org.apache.commons.lang3.ClassUtils
2020-08-14T07:01:50.3890718Z [INFO] [ERROR] -> [Help 1]
2020-08-14T07:01:50.3890909Z [INFO] [ERROR] 
2020-08-14T07:01:50.3891133Z [INFO] [ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
2020-08-14T07:01:50.3891396Z [INFO] [ERROR] Re-run Maven using the -X switch to enable full debug logging.
2020-08-14T07:01:50.3891614Z [INFO] [ERROR] 
2020-08-14T07:01:50.3891832Z [INFO] [ERROR] For more information about the errors and possible solutions, please read the following articles:
2020-08-14T07:01:50.3892116Z [INFO] [ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/PluginContainerException
2020-08-14T07:01:50.5731409Z [INFO]   The build exited with code 1. See D:\a\dropwizard\dropwizard\dropwizard-archetypes\java-simple\target\it\projects\ordinary\test-project\build.log for details.
2020-08-14T07:01:50.5733585Z [INFO]           ordinary\test-project\pom.xml .................... FAILED (19.7 s)
```

--- a/.github/workflows/maven.yml
+++ b/.github/workflows/maven.yml
@@ -37,7 +37,7 @@ jobs:
         key: ${{ runner.os }}-sonar
         restore-keys: ${{ runner.os }}-sonar
     - name: Build
-      run: ./mvnw --no-transfer-progress -V -B -ff -s .github/settings.xml -e "-DtrimStackTrace=false" "-Dmaven.test.failure.ignore=true" "-Dsurefire.rerunFailingTestsCount=1" verify
+      run: ./mvnw --no-transfer-progress -V -B -ff -s .github/settings.xml -e "-DtrimStackTrace=false" "-Dmaven.test.failure.ignore=true" "-Dsurefire.rerunFailingTestsCount=1" install
     - name: Publish Test Results
       # If the CI run is not initiated from the primary repository, it is highly likely that this is a PR from a user who doesn't have commit rights.
       # Hence, skip this step to avoid permission failures.

--- a/.github/workflows/maven.yml
+++ b/.github/workflows/maven.yml
@@ -15,7 +15,7 @@ jobs:
       fail-fast: false
       matrix:
         java_version: ['8', '11', '17']
-        os: ['ubuntu-latest']
+        os: ['ubuntu-latest', 'windows-latest']
     env:
       JAVA_OPTS: "-XX:+TieredCompilation -XX:TieredStopAtLevel=1"
     steps:

--- a/.github/workflows/maven.yml
+++ b/.github/workflows/maven.yml
@@ -37,7 +37,7 @@ jobs:
         key: ${{ runner.os }}-sonar
         restore-keys: ${{ runner.os }}-sonar
     - name: Build
-      run: ./mvnw --no-transfer-progress -V -B -ff -s .github/settings.xml -e -DtrimStackTrace=false -Dmaven.test.failure.ignore=true -Dsurefire.rerunFailingTestsCount=1 verify
+      run: ./mvnw --no-transfer-progress -V -B -ff -s .github/settings.xml -e "-DtrimStackTrace=false" "-Dmaven.test.failure.ignore=true" "-Dsurefire.rerunFailingTestsCount=1" verify
     - name: Publish Test Results
       # If the CI run is not initiated from the primary repository, it is highly likely that this is a PR from a user who doesn't have commit rights.
       # Hence, skip this step to avoid permission failures.
@@ -60,4 +60,4 @@ jobs:
       env:
         GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}  # Needed to get PR information, if any
         SONAR_TOKEN: ${{ secrets.SONAR_TOKEN }}
-      run: ./mvnw --no-transfer-progress -V -B -ff -s .github/settings.xml -Dsonar.projectKey=dropwizard_dropwizard -Dsonar.organization=dropwizard -Dsonar.host.url=https://sonarcloud.io org.sonarsource.scanner.maven:sonar-maven-plugin:sonar
+      run: ./mvnw --no-transfer-progress -V -B -ff -s .github/settings.xml "-Dsonar.projectKey=dropwizard_dropwizard" "-Dsonar.organization=dropwizard" "-Dsonar.host.url=https://sonarcloud.io" org.sonarsource.scanner.maven:sonar-maven-plugin:sonar

--- a/dropwizard-client/src/test/java/io/dropwizard/client/DropwizardSSLConnectionSocketFactoryTest.java
+++ b/dropwizard-client/src/test/java/io/dropwizard/client/DropwizardSSLConnectionSocketFactoryTest.java
@@ -14,8 +14,6 @@ import org.glassfish.jersey.client.ClientResponse;
 import org.junit.jupiter.api.AfterAll;
 import org.junit.jupiter.api.BeforeEach;
 import org.junit.jupiter.api.Test;
-import org.junit.jupiter.api.condition.DisabledOnOs;
-import org.junit.jupiter.api.condition.OS;
 import org.junit.jupiter.api.extension.ExtendWith;
 
 import javax.net.ssl.HostnameVerifier;
@@ -153,7 +151,6 @@ class DropwizardSSLConnectionSocketFactoryTest {
     }
 
     @Test
-    @DisabledOnOs(OS.WINDOWS) // https://github.com/dropwizard/dropwizard/issues/4330
     void shouldReturn200IfAbleToClientAuth() throws Exception {
         tlsConfiguration.setKeyStorePath(toFile("stores/client/keycert.p12"));
         tlsConfiguration.setKeyStorePassword("password");
@@ -176,7 +173,6 @@ class DropwizardSSLConnectionSocketFactoryTest {
     }
 
     @Test
-    @DisabledOnOs(OS.WINDOWS) // https://github.com/dropwizard/dropwizard/issues/4330
     void shouldReturn200IfAbleToClientAuthSpecifyingCertAliasForGoodCert() throws Exception {
         tlsConfiguration.setKeyStorePath(toFile("stores/client/twokeys.p12"));
         tlsConfiguration.setKeyStorePassword("password");
@@ -263,7 +259,6 @@ class DropwizardSSLConnectionSocketFactoryTest {
     }
 
     @Test
-    @DisabledOnOs(OS.WINDOWS) // https://github.com/dropwizard/dropwizard/issues/4330
     void shouldSucceedWithBcProvider() throws Exception {
         // switching host verifier off for simplicity
         tlsConfiguration.setVerifyHostname(false);

